docs/2023-02-14-so-whats-next.md

```diff
@@ -211,45 +211,45 @@ The best way forward for JavaScript would be for TC39 and `core-js` to work toge
 
-### Some words about war
+### Some words about the war
 
 **Open-source should be out of politics.**
 
 I don't want to choose between two kinds of evil. I will not comment on this in more detail, since there are people close to me on both sides of the border who may suffer because of this.
 
-Let me remind you what I wrote about above - I returned to Russia because it was a place where it was possible to have a decent standard of living for relatively small money and concentrate on FOSS instead of making money. Now I cannot leave Russia, because after the accident I have outstanding lawsuits in the amount of tens of thousands of dollars and I am forbidden to leave the country until they are paid off.
+Let me remind you what I wrote about above: I returned to Russia because it was a place where it was possible to have a decent standard of living for relatively small money and to concentrate on FOSS instead of making money. Now I cannot leave Russia, because after the accident I have outstanding lawsuits in the amount of tens of thousands of dollars and I am forbidden to leave the country until they are paid off.
 
-### What do you think, how much money `core-js` earn for a month?
+### What do you think, how much money `core-js` earns each month?
 
-When I started to maintain `core-js` full-time, without being distracted by contracts and any other work, **it was about \$2500 for a month - it was about 4-5 times less than I usually had on full-time contracts**. Remember, a kind of down-shifting, to make the Web better. Temporarily. Reduce issues and bugs to zero, make the highest quality product, which is used by almost everyone - and the project will be sufficiently supported, right? Right?
+When I started to maintain `core-js` full-time, without being distracted by contracts and any other work, **it was about \$2500 per month - it was about 4-5 times less than I usually had on full-time contracts**. Remember, a kind of down-shifting, to make the Web better. Temporarily. Reduce issues and bugs to zero, make the highest quality product, which is used by almost everyone... and the project will be sufficiently supported, right? Right?
 
-After a few months, monthly repeat was **decreased to about \$1700** *(at least that's what I thought)* - \$1000 via Tidelift, \$600 via Open Collective, and \$100 via Patreon. In addition to the monthly repeat, one-time donations periodically come - on average it was maybe \$100 for a month.
+After a few months, the reoccurring monthly income **decreased to about \$1700** *(at least that's what I thought)*, \$1000 via Tidelift, \$600 via Open Collective, and \$100 via Patreon. In addition to the reoccurring monthly, one-time donations come periodically (on average it was maybe \$100 in a month).
```

Comment:
Are you sure about "reoccurring monthly" instead of "monthly repeat"? I heard the second more often in this case.